### ISSUE TITLE
Project abandonated?

### ISSUE THREAD

**Issue by pedrxd** [Lectem/3Damnesic#3]

**Comment by Lectem** [comment]
Yeah it's abandoned, but if some people want to resurect it I can help a bit

**Comment by pedrxd** [comment]
Okey, it will be fine if someone continue it. But probably is not necesary.

**Comment by DrHacknik** [comment]
I may. I've been thinking about it, and i may be able to get some VLC components in.
